### PR TITLE
Bump npm version correctly

### PR DESCRIPTION
**Pull request by habemuscode**
**Related to**

https://github.com/oat-sa/extension-tao-itemqti/pull/1426

**Description**

Bump version to the real number

### CODE REVIEW

File: views/package.json [hunk 1]
```diff
@@ -9,6 +9,6 @@
   },
   "dependencies": {
     "@oat-sa/tao-item-runner": "0.4.0",
-    "@oat-sa/tao-item-runner-qti": "github:oat-sa/tao-item-runner-qti-fe#fix/BOSA-36/text-entry-pattermask"
+    "@oat-sa/tao-item-runner-qti": "0.4.19"
```
Comment (wazelin) [inline views/package.json:12]: I don't get it.
The [`develop` branch](https://github.com/oat-sa/extension-tao-itemqti/blob/develop/views/package.json) seem to have it already, how come it shows that diff?

File: views/package-lock.json [hunk 2]
```diff
@@ -10,9 +10,9 @@
       "integrity": "sha512-Z/fRV8TSxitZwO40zeL8T+pLbxoqp5L2dDqWf+0kPOV3Guf2juP1eXZygiXhwmHHOTgwpSeKSC0sOJJATsL5xQ=="
     },
     "@oat-sa/tao-item-runner-qti": {
-      "version": "0.4.18",
-      "resolved": "https://registry.npmjs.org/@oat-sa/tao-item-runner-qti/-/tao-item-runner-qti-0.4.18.tgz",
-      "integrity": "sha512-gLiMNqsGSs82UhtCuYIdsCf1IA94sHIyEsBrKKyb90sJF++W5pw8+2xb95+RGmB+k+R1cnjDj5IgsIejAswyOw=="
+      "version": "0.4.19",
+      "resolved": "https://registry.npmjs.org/@oat-sa/tao-item-runner-qti/-/tao-item-runner-qti-0.4.19.tgz",
+      "integrity": "sha512-scjmtp1LEZyyhzwz5LKjZ/0BO47cC3KjWrzoZGfaAKZW3bn8zJpbTCI5aDQDOD8A3FPBfHY0IFs87jsHyG4CWA=="
```
Comment (wazelin) [inline views/package-lock.json:15]: Same [here](https://github.com/oat-sa/extension-tao-itemqti/blob/develop/views/package-lock.json).

File: views/package.json (hunk 1 above)
Comment (habemuscode) [reply views/package.json]: Npm package was release to npm repository, so its better use this way. Apart of that, I deploy on playground and dont see the new code  (I dont know if its for this )

**Comment by jsconan** [comment]
I merged develop in it, and... the PR is empty now. I guess you had an outdated develop branch locally.

**Comment by wazelin** [comment]
That explains it. Right, Github shows the commit diffs, not the branch diffs.
Closing the PR then.

File: views/package.json (hunk 1 above)
Comment (wazelin) [reply views/package.json]: I'll check the deployment for you, don't worry.

**Pull request closed without merging**